tests/specs/check/net_types/__test__.jsonc

```diff
@@ -0,0 +1,16 @@
+{
+  "steps": [
+    {
+      "args": "check jsr:@std/http",
+      "output": "http.out"
+    },
+    {
+      "args": "check https://deno.land/x/fresh/src/types.ts",
```

Comment:
> @dsherret, I want to make actual JSR requests here. Should the spec tests allow me to do this right now?

We should make the tests work without an internet connection.

Reply:
We could have these codebases as git submodules, but I don't think that's desirable. I assume we'd like to keep git submodules to a minimum.

@mmastrac, I wonder if we should even do this or instead keep a mental note for the rare scenario that we change network types to do this manually.